testlist.sh

```diff
@@ -1,20 +1,172 @@
+MDT_suite=( [16Clnt_MDTeasy_SX]="1 256 16 32 SX 300 NO" [16Clnt_MDThard_SX]="1 256 16 32 SX 300 NO"
+            [16Clnt_MDTeasy_2GX]="2 256 16 32 RP_2GX 300 NO" [16Clnt_MDThard_2GX]="2 256 16 32 RP_2GX 300 NO"
+            [16Clnt_MDTeasy_3GX]="4 256 16 32 RP_3GX 300 NO" [16Clnt_MDThard_3GX]="4 256 16 32 RP_3GX 300 NO"
+            [1Sto4C_MDTeasy_SX]="1 256 16 32 SX 60 NO" [1Sto4C_MDThard_SX]="1 256 16 32 SX 60 NO"
+            [1Sto4C_MDTeasy_2GX]="2 256 16 32 RP_2GX 60 NO" [1Sto4C_MDThard_2GX]="2 256 16 32 RP_2GX 60 NO"
+            [1Sto4C_MDTeasy_3GX]="4 256 16 32 RP_3GX 60 NO" [1Sto4C_MDThard_3GX]="4 256 16 32 RP_3GX 60 NO" )
+
+# Set the needed PATH and LD_LIBRARY_PATH
 export PATH=/opt/apps/xalt/xalt/bin:/opt/apps/intel19/python3/3.7.0/bin:/opt/apps/cmake/3.16.1/bin:/opt/apps/autotools/1.2/bin:/opt/apps/git/2.24.1/bin:/opt/intel/compilers_and_libraries_2019.5.281/linux/bin/intel64:/opt/apps/gcc/8.3.0/bin:/usr/lib64/qt-3.3/bin:/usr/local/bin:/bin:/usr/bin:/opt/ibutils/bin:/opt/ddn/ime/bin:/opt/dell/srvadmin/bin:.
 export LD_LIBRARY_PATH=/opt/apps/intel19/python3/3.7.0/lib:/opt/intel/debugger_2019/libipt/intel64/lib:/opt/intel/compilers_and_libraries_2019.5.281/linux/daal/lib/intel64_lin:/opt/intel/compilers_and_libraries_2019.5.281/linux/tbb/lib/intel64_lin/gcc4.7:/opt/intel/compilers_and_libraries_2019.5.281/linux/mkl/lib/intel64_lin:/opt/intel/compilers_and_libraries_2019.5.281/linux/ipp/lib/intel64:/opt/intel/compilers_and_libraries_2019.5.281/linux/compiler/lib/intel64_lin:/opt/apps/gcc/8.3.0/lib64:/opt/apps/gcc/8.3.0/lib:/usr/lib64/:/usr/lib64/
 
-DAOS_DIR="<path_to_daos>"
-DST_DIR="<path_to_daos_scaled_testing>"
-RES_DIR="<path_to_result_dir>"
-JOBNAME="<sbatch_jobname>"
-TIMEOUT="<sbatch_timeout>" #<hh:mm:ss>
-EMAIL="<email>" #<first.last@intel.com>
+# Define the location od daos, sacled testing, results. Extract the build identity as work week identity
+DAOS_DIR="/work/06758/arunar/frontera/BUILDS/20WW33.3+eccd948/daos"
+DST_DIR="/work/06758/arunar/frontera/TESTS/daos_scaled_testing"
+RES_DIR="/work/06758/arunar/frontera/WEEKLY_RESULTS"
+sub_str=${DAOS_DIR%/*}
+if [[ -L "$sub_str" ]] ; then
+    dir_str=`ls -la $dir_str | awk '{print $(NF)}'`
+else
+    dir_str=${sub_str##*/}
+fi
+BLD_DIR=${dir_str:0:8}
+
+# Adjust the TIMEOUT tomaximum expected run time for any test
+JOBNAME="scaletest"
+TIMEOUT="2:00:00" #<hh:mm:ss>
+EMAIL="aruna.ramanan@intel.com" #<first.last@intel.com>
 
 SBPARAMS="-J $JOBNAME -t $TIMEOUT --mail-user=$EMAIL"
 
 export DAOS_DIR
 pushd $DST_DIR
 
+
+# For each test config in the suite run the IOR tests doubling the number of servers each time from min to max servers
+run_IOR () {
+for  element in "${!IOR_suite[@]}"; do 
+
+    export TESTCASE=$element
+    export LOGS=$RES_DIR/$BLD_DIR/$TESTCASE
+    echo $LOGS
+    mkdir -p $LOGS
+
+    # Extract the test caregory and IOR test type. 
+    IFS='_' ; read -ra tcase <<<"$element" 
```

Comment:
I would prefer if the test behavior is not encoded in the name. It makes it difficult to add a new testcase. you need to (1) comply to a certain naming convention (2) add another block of code to the if/else at line 92